samples/matter/lock/VERSION

```diff
@@ -3,3 +3,13 @@ VERSION_MINOR = 7
 PATCHLEVEL = 99
 VERSION_TWEAK = 0
 EXTRAVERSION =
+
+# SUIT configuration
+APP_ROOT_SEQ_NUM = 1
+APP_ROOT_VERSION = 0.1.0
+APP_LOCAL_1_SEQ_NUM = 1
+APP_LOCAL_1_VERSION = 0.1.0
+APP_RECOVERY_SEQ_NUM = 1
+APP_RECOVERY_VERSION = 0.1.0
+RAD_LOCAL_1_SEQ_NUM = 1
+RAD_LOCAL_1_VERSION = 0.1.0
```

Comment:
Just FYI - depending on your needs, it is possible to skip it and allow the SUIT generator to use Zephyr version values to populate SUIT versions/sequence numbers.

Reply:
Do I need to set the additional config, or just remove those lines from there?

Reply:
If you want to use the defaults - it is enough ti remove them :slightly_smiling_face: